### ISSUE TITLE
Use Bottom Sheet dialog for sharing image

### ISSUE THREAD

**Issue by pa1pal** [fossasia/phimpme-android#881]
**Actual Behaviour**

Currently we are using a separate `sharingAcitivity` for sharing images, to different accounts. We can use the BottomSheetDialog which is generally used for multiple sharing options.

**Expected Behaviour**

Showing BottomSheetDialog as a sharing option with the same user interface. 

**Would you like to work on the issue?**

Yes

**Comment by vinaysajjanapu** [comment]
@pa1pal I like the current way of sharing.
@mohitmanuja @anantprsd5 @Subhankar29 please say your opinion.

**Comment by pa1pal** [comment]
@vinaysajjanapu The point is what is designed for what. BottomSheet Dialog widely used for sharing purposes because of its quick actions and swipe navigation. The layout will be exactly same as of now, it will just appear from the bottom.

One thing I noticed now is, in case of our app, the sharing activity invoke from multiple location, there it will hinder. Will think of unification of this feature. Closing this issue for now.
